CP02PMCI/CP02PMCI_R00011_ingest.csv

```diff
@@ -0,0 +1,11 @@
+parser,filename_mask,reference_designator,data_source,status,notes
+mi.dataset.driver.adcps_jln.stc.adcps_jln_stc_recovered_driver,/omc_data/whoi/OMC/CP02PMCI/R00011/cg_data/stc1/imm/adcpt/adcpt*.DAT,CP02PMCI-RII01-02-ADCPTG010,recovered_host,Expected,
+mi.dataset.driver.adcps_jln.adcps_jln_driver,/omc_data/whoi/OMC/CP02PMCI/R00011/instruments/ADCP*/*RDI_000.000,CP02PMCI-RII01-02-ADCPTG010,recovered_inst,Expected,
+mi.dataset.driver.cg_stc_eng.stc.cg_stc_eng_stc_recovered_driver,/omc_data/whoi/OMC/CP02PMCI/R00011/cg_data/stc1/irid2shore/stc_status*.txt,CP02PMCI-SBS01-00-STCENG000,recovered_host,Expected,
+mi.dataset.driver.cg_stc_eng.stc.mopak_o_dcl_recovered_driver,/omc_data/whoi/OMC/CP02PMCI/R00011/cg_data/stc1/3dmgx3/*.3dmgx3.log,CP02PMCI-SBS01-01-MOPAK0000,recovered_host,Expected,
+mi.dataset.driver.WFP_ENG.STC_IMODEM.wfp_eng_stc_imodem_recovered_driver,/omc_data/whoi/OMC/CP02PMCI/R00011/instruments/CWFP*/0*00/E*.DA?,CP02PMCI-WFP01-00-WFPENG000,recovered_wfp,Expected,
+mi.dataset.driver.vel3d_k.wfp.vel3d_k_wfp_recovered_driver,/omc_data/whoi/OMC/CP02PMCI/R00011/instruments/CWFP*/0*00/A*.DA?,CP02PMCI-WFP01-01-VEL3DK000,recovered_wfp,Expected,
+mi.dataset.driver.dofst_k.wfp.dofst_k_wfp_recovered_driver,/omc_data/whoi/OMC/CP02PMCI/R00011/instruments/CWFP*/0*00/C*.DA?,CP02PMCI-WFP01-02-DOFSTK000,recovered_wfp,Expected,
+mi.dataset.driver.ctdpf_ckl.wfp.ctdpf_ckl_wfp_recovered_driver,/omc_data/whoi/OMC/CP02PMCI/R00011/instruments/CWFP*/0*00/C*.DA?,CP02PMCI-WFP01-03-CTDPFK000,recovered_wfp,Expected,
+mi.dataset.driver.flort_kn.stc_imodem.flort_kn__stc_imodem_recovered_driver,/omc_data/whoi/OMC/CP02PMCI/R00011/instruments/CWFP*/0*00/E*.DA?,CP02PMCI-WFP01-04-FLORTK000,recovered_wfp,Expected,
+mi.dataset.driver.PARAD_K.STC_IMODEM.parad_k_stc_imodem_recovered_driver,/omc_data/whoi/OMC/CP02PMCI/R00011/instruments/CWFP*/0*00/E*.DA?,CP02PMCI-WFP01-05-PARADK000,recovered_wfp,Expected,
```

Comment:
All the WFP lines have ".DA?" instead of ".DAT".

Reply:
In rare instances the files are *.DAC, so the parser needs to handle both types.